src/util/index.js

```diff
@@ -36,7 +36,7 @@ export const setupCacheFile = () => {
       console.log(`Successfully created ${oneAppDirectoryPath}`);
       console.log(`Creating ${cacheFileName}`);
       try {
-        fs.writeFileSync(oneAppModuleCachePath, JSON.stringify('{}'));
+        fs.writeFileSync(oneAppModuleCachePath, JSON.stringify({}));
```

Comment:
this needs a test

Reply:
Done [e42e29e00e92a4017cba283a2d593b80ea2dfec1](https://github.com/americanexpress/one-app-dev-cdn/pull/51/commits/e42e29e00e92a4017cba283a2d593b80ea2dfec1)